lib/sidekiq.rb

```diff
@@ -149,7 +149,12 @@ def self.default_server_middleware
   end
 
   def self.default_worker_options=(hash)
-    @default_worker_options = default_worker_options.merge(hash.stringify_keys)
+    h = hash.dup
+    # stringify
+    h.keys.each do |key|
+      h[key.to_s] = h.delete(key)
+    end
+    @default_worker_options = default_worker_options.merge(h)
```

Comment:
I don't know why some people seem to have a hard time trying to understand why the existence of symbols is what I consider Ruby's worst part ;)